cmd/host/src/lib.rs

```diff
@@ -77,35 +78,114 @@ struct HostArgs {
     cmd: HostCommand,
 }
 
-fn read_var(
+static SEPARATE_HOST_BOOT_FAIL_NAME: &str = "LAST_HOST_BOOT_FAIL";
+
+static SEPARATE_LAST_HOST_PANIC_NAME: &str = "LAST_HOST_PANIC";
+
+static HOST_STATE_BUF_NAME: &str =
+    "task_host_sp_comms::ServerImpl::claim_static_resources::BUFS";
```

Comment:
i hope the buffer this is referencing never changes names

Reply:
Separately, I'm preparing a commit in Hubris that adds a big freakin' comment to that buffer saying something like

```rust
// HEY
//
// HEY LISTEN
//
// HUMILITY DEPENDS ON THE NAME OF THIS VARIABLE
```